schema.sql

```diff
@@ -10,3 +10,46 @@ CREATE TABLE animals (
 
 -- ADD Column Species of type string
  ALter table animals ADD species varchar(1000);
+
+-- Task 3
+
+/*
+Create a table named owners with the following columns:
+    id: integer (set it as autoincremented PRIMARY KEY)
+    full_name: string
+    age: integer
+*/
+
+CREATE TABLE owners (
+    id int NOT NULL GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    full_name  varchar(1000),
+    age  int 
+);
+
+/*
+
+Create a table named species with the following columns:
+    id: integer (set it as autoincremented PRIMARY KEY)
+    name: string
+*/
```

Comment:
- [ ] Kindly delete all block comments because they make the code unreadable

Reply:
I used these comments for later reference of my code. i thought they make the code abit more meaningful.